packages/terra-paginator/src/Paginator.module.scss

```diff
@@ -33,14 +33,12 @@
       max-width: 100%; /* Need to help with overflow-wrap */
       min-width: var(--terra-paginator-nav-link-min-width, 2rem); // Helps ensure limited resizing as you page through large numbers.
       outline: none;
-      overflow-wrap: break-word; /* Modern browsers */
       padding: var(--terra-paginator-nav-link-padding, 0.35714rem 0.5rem);
       text-align: center;
       text-decoration: var(--terra-paginator-nav-link-text-decoration, none);
       touch-action: manipulation; // Enable fast tap interaction in webkit browsers; see https://webkit.org/blog/5610/more-responsive-tapping-on-ios/
       user-select: none; // Keeps user from highlighting and text selection
       white-space: normal;
-      word-wrap: break-word; /* For IE 10 and IE 11 */
```

Comment:
I'd like to confirm with UX they are okay with the next and previous buttons wrapping on Chrome:

![image](https://user-images.githubusercontent.com/6720991/77322951-83ed5800-6ce2-11ea-8ba6-9fcb4bec299b.png)

On IE 10 the "Last" button can escape the bounds of the container:


![image](https://user-images.githubusercontent.com/6720991/77323255-f5c5a180-6ce2-11ea-97a9-2762642d1f8d.png)

Reply:
It's not ideal. I didn't notice that happening when I looked at it initially. Removing the seal of approval until after this gets addressed.

Reply:
I think the wrapping in chrome is the correct function, but it is not happening in IE10 due to which the `Last` button goes out... Any css inputs  to fix it in IE10? cc/ @jeremyfuksa .

Reply:
`white-space: nowrap` gets the buttons to do what I think should be happening in Chrome. Removing `min-width` of `.nav-link` in IE 10 get the right spacing.
![Screen Shot 2020-04-07 03 27 45 PM](https://user-images.githubusercontent.com/726351/78716325-590b2280-78e4-11ea-91e4-e187358d1629.png)

Reply:
Changing `background-size: auto` for `.icon` in IE 10 also keeps the chevrons from being cut off.

Reply:
Updated 9ab38f7